tests/integration/tests/services_test.go

```diff
@@ -24,12 +24,14 @@ func TestServicesList(t *testing.T) {
 	require.True(len(serviceList.Services) >= 4)
 	for _, service := range serviceList.Services {
 		require.NotEmpty(service.Name)
-		require.True(service.IstioSidecar)
-		require.True(service.AppLabel)
-		require.NotNil(service.Health)
-		require.NotNil(service.Health.Requests)
-		require.NotNil(service.Health.Requests.Outbound)
-		require.NotNil(service.Health.Requests.Inbound)
+		if service.Name == "productpage" {
+			require.True(service.IstioSidecar)
+			require.True(service.AppLabel)
+			require.NotNil(service.Health)
+			require.NotNil(service.Health.Requests)
+			require.NotNil(service.Health.Requests.Outbound)
+			require.NotNil(service.Health.Requests.Inbound)
```

Comment:
```suggestion
			require.NotNil(service.Health.Requests.Inbound)
			break
```

It would be more performant with a break but the service list is not very large so it is non-blocking issue.

Reply:
we still need to check other services at least the name